src/partition/repart.c

```diff
@@ -1591,6 +1592,13 @@ static int context_load_partition_table(
         if (r < 0)
                 return log_error_errno(r, "Failed to open device '%s': %m", node);
 
+        /* FIXME: it's probably better to take this out of its current caller
```

Comment:
Don't add a FIXME comment when you're gonna remove it some commit later.

Reply:
Probably I should just squash commit 1 and 3.